src/rocketData.py

```diff
@@ -196,7 +196,16 @@ def convert_to_csv(self)
             self._data['encoders']['position'],
             self._data['time_stamp']
         ]
-        return dataToConvert
+        return allRocketData
+
+    def convert_to_csv_string(self):
+        dToConvert = self.all_rocket_data()
+        csvString = str(dToConvert[0])
+        for d in dToConvert[1:]:
+            csvString += "," + str(d)
```

Comment:
```suggestion
            csv_string += ',  ${d}'.format(d = str(data))
```
This might not be necessary not sure how they do it in python land. But for es6 and C# you normally want to format strings without `+` concatenation.

Reply:
I don't think there really is an efficient way to build strings in python. I did find this [article](https://waymoot.org/home/python_string/) about it, but I'm not sure if it's necessary. Also, that line doesn't really work without using string templates.
csv_string += Template(', ${d}').substitute(d = str(data))

Reply:
cool its probably fine this way then